### PR TITLE
Update NLPSandboxClient (dependency) Version

### PR DESCRIPTION
**Pull request by cascadianblue**
Makes some changes in the deidentifier codebase to accomodate this version update. Does the work of #115 .

### CODE REVIEW

File: server/openapi_server/controllers/deidentified_note_controller.py [hunk 1]
```diff
@@ -48,30 +48,31 @@ def create_deidentified_notes(deidentify_request=None):  # noqa: E501
         if 'text_date' in all_annotation_types:
             annotations['text_date'] = client.annotate_note(
                 host=config.date_annotator_api_url,
-                note=client_note, tool_type='nlpsandbox:date-annotator')[
+                note=client_note, tool_type='nlpsandbox:date-nlpsandboxsdk')[
```
Comment (tschaffter) [inline server/openapi_server/controllers/deidentified_note_controller.py:51]: This is not a valid type. See comment below.

File: server/openapi_server/test/utils.py [hunk 2]
```diff
@@ -18,13 +18,14 @@ def client_note_to_request_dict(note: Note):
 
 
 ANNOTATOR_TYPE_MAP = {
-    'nlpsandbox:physical-address-annotator': (
+    'nlpsandbox:physical-address-nlpsandboxsdk': (
```
Comment (tschaffter) [inline server/openapi_server/test/utils.py:21]: Not a valid too type.

**Review by tschaffter — CHANGES_REQUESTED** [review]

File: server/openapi_server/test/utils.py (hunk 2 above)
Comment (cascadianblue) [reply server/openapi_server/test/utils.py:21]: I thought it was weird, too, but it appears to be [what `nlpsandboxclient` expects](https://github.com/nlpsandbox/nlpsandbox-client/blob/d58e400a5c57d8fe4b4ae891fe4b5bc35b9201af/nlpsandboxclient/client.py#L503).

**Comment by tschaffter** [comment]
@cascadianblue See tickets listed above this message. We will fix the client, re-release its v2 that we made yesterday and then I will come back to this ticket.

File: server/openapi_server/test/utils.py (hunk 2 above)
Comment (thomasyu888) [reply server/openapi_server/test/utils.py:21]: Great find guys!  Aggressive search and find issue strikes again.

**Comment by thomasyu888** [comment]
@cascadianblue, @tschaffter ,  `nlpsandbox-client` 3.0.0 has been released

**Comment by tschaffter** [comment]
@cascadianblue @thomasyu888 Update is done in #115

**Pull request closed without merging**